rpc_jobs/unit/single_use_slave.yml

```diff
@@ -14,6 +14,10 @@
           REGIONS: "DFW ORD"
           FALLBACK_REGIONS: "IAD"
       - rpc_gating_params
+      - string:
+          name: SLAVE_TYPE
+          default: container
```

Comment:
You could use the `standard_job_params` params here instead.  The default there is `instance`.

Reply:
Ah good plan.

Reply:
Adjusted this.